core/src/com/unciv/ui/options/GameplayTab.kt

```diff
@@ -33,4 +33,5 @@ fun gameplayTab(
         settings.automatedWorkersReplaceImprovements
     ) { settings.automatedWorkersReplaceImprovements = it }
     optionsPopup.addCheckbox(this, "Order trade offers by amount", settings.orderTradeOffersByAmount) { settings.orderTradeOffersByAmount = it }
-}
\ No newline at end of file
+    optionsPopup.addCheckbox(this, "Ask for confirmation when pressing next turn", settings.confirmNextTurn) { settings.confirmNextTurn = it }
```

Comment:
Add to template.properties. Make sure your editor doesn't ruin the trailing blank.